ENIGMAsystem/SHELL/Graphics_Systems/General/GSvertex.h

```diff
@@ -18,52 +19,74 @@
 #ifndef ENIGMA_GSVERTEX_H
 #define ENIGMA_GSVERTEX_H
 
-#include "../General/OpenGLHeaders.h"
-#include <vector>
-#include <utility>
-using std::vector;
-using std::pair;
-
 #include "Universal_System/scalar.h"
 
-namespace enigma {
-  struct VertexFormat {
-    vector<pair<int,int> > flags;
+namespace enigma_user {
 
-    void AddAttribute(int type, int attribute) {
-      flags.push_back(std::make_pair(type, attribute));
-    }
-  };
-}
+enum {
+  vertex_type_float1, // D3DDECLTYPE_FLOAT1
+  vertex_type_float2, // D3DDECLTYPE_FLOAT2
+  vertex_type_float3, // D3DDECLTYPE_FLOAT3
+  vertex_type_float4, // D3DDECLTYPE_FLOAT4
+  vertex_type_color,  // D3DDECLTYPE_D3DCOLOR
+  vertex_type_ubyte4  // D3DDECLTYPE_UBYTE4
+};
 
-namespace enigma_user {
-  enum {
-    vertex_type_float1, // D3DDECLTYPE_FLOAT1
-    vertex_type_float2, // D3DDECLTYPE_FLOAT2
-    vertex_type_float3, // D3DDECLTYPE_FLOAT3
-    vertex_type_float4, // D3DDECLTYPE_FLOAT4
-    vertex_type_colour, // D3DDECLTYPE_D3DCOLOR
-    vertex_type_ubyte4  // D3DDECLTYPE_UBYTE4
-  };
+#define vertex_type_colour vertex_type_color
```

Comment:
can use a using here instead

Reply:
I'm not prepared to deal with this problem right now. Josh said to just use a `#define` because the other way with constexpr won't work with functions "in general" and LGM can't show them in its autocomplete list. I can look at moving all of these colour overloads and aliases into a British extension later in a future PR or coming up with a more advanced technique of dealing with this.